packages/host/app/commands/create-ai-assistant-room.ts

```diff
@@ -0,0 +1,60 @@
+import { service } from '@ember/service';
+
+import format from 'date-fns/format';
+
+import { aiBotUsername } from '@cardstack/runtime-common';
+
+import type * as BaseCommandModule from 'https://cardstack.com/base/command';
+
+import HostBaseCommand from '../lib/host-base-command';
+
+import { AI_BOT_POWER_LEVEL } from '../services/matrix-service';
+
+import type MatrixService from '../services/matrix-service';
+
+export default class CreateAIAssistantRoomCommand extends HostBaseCommand<
+  BaseCommandModule.CreateAIAssistantRoomInput,
+  BaseCommandModule.CreateAIAssistantRoomResult
+> {
+  @service private declare matrixService: MatrixService;
+
+  async getInputType() {
+    let commandModule = await this.loadCommandModule();
+    const { CreateAIAssistantRoomInput } = commandModule;
+    return CreateAIAssistantRoomInput;
+  }
+
+  protected async run(
+    input: BaseCommandModule.CreateAIAssistantRoomInput,
+  ): Promise<BaseCommandModule.CreateAIAssistantRoomResult> {
+    let { matrixService } = this;
+    let { matrixSDK, userId } = matrixService;
+    if (!userId) {
+      throw new Error(
+        `bug: there is no userId associated with the matrix client`,
+      );
+    }
+    let server = userId!.split(':')[1];
+    let aiBotFullId = `@${aiBotUsername}:${server}`;
+    let { room_id: roomId } = await matrixService.createRoom({
+      preset: matrixSDK.Preset.PrivateChat,
```

Comment:
consider making matrixService expose just the PrivateChat const rather than all of matrixSDK